app/build.gradle.kts

```diff
@@ -57,11 +57,18 @@ dependencies {
     implementation(libs.androidx.activity.compose)
     implementation(platform(libs.androidx.compose.bom))
     implementation(libs.androidx.ui)
+    implementation(libs.graphics.shapes)
```

Comment:
Where do you use this dependency? If you don't use it anywhere it is safe to delete it